src/connection.rs

```diff
@@ -152,7 +154,10 @@ async fn requests(topic: String, payload_size: usize, count: usize, requests_tx:
 
         let publish = PublishRaw::new(&topic, qos, payload).unwrap();
         let publish = Request::PublishRaw(publish);
-        requests_tx.send(publish).await.unwrap();
+        interval.tick().await;
```

Comment:
Nice!! I would have used `time::delay_for`. But I think `tick` is the right thing to use here.

Reply:
```.  If the tick in the example below was replaced with delay_for, the task would only be executed once every three seconds, and not every two seconds.
```

`tick` and `interval` stream is kinda more appropriate as it will trigger the task without caring about time elapsed between tasks.

Reply:
Yeah